Sources/NIO/SelectorKqueue.swift

```diff
@@ -0,0 +1,356 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2021 Apple Inc. and the SwiftNIO project authors
```

Comment:
simliar note here, I think 2017-2021 is correct

Reply:
Fixed in e19e685.